sharktank_models/llama3.1/test_llama.py

```diff
@@ -0,0 +1,111 @@
+import iree.compiler
+import iree.runtime
+import numpy
+import os
+import pytest
+import torch
+
+page_size = 12288
+block_size = 16
+
+
+@pytest.fixture
+def llama_mlir():
+    dirname = os.path.dirname(__file__)
+    return f"{dirname}/assets/toy_llama.mlir"
+
+
+@pytest.fixture
+def llama_irpa():
+    dirname = os.path.dirname(__file__)
+    return f"{dirname}/assets/toy_llama.irpa"
+
+
+@pytest.fixture
+def compile_llama_cpu(llama_mlir):
+    return iree.compiler.compile_file(
+        llama_mlir,
+        extra_args=[
+            "--iree-hal-target-device=llvm-cpu",
+            "--iree-llvmcpu-target-cpu=host",
+        ],
+    )
+
+
+@pytest.fixture
+def compile_llama_gfx1100(llama_mlir):
+    return iree.compiler.compile_file(
+        llama_mlir,
+        extra_args=["--iree-hal-target-device=hip", "--iree-hip-target=gfx1100"],
+    )
```

Comment:
Could load the target from an environment variable if we want to be able to run the tests on different GPUs without needing to edit the source.

Reply:
Added environment variable HIP device